src/overview.md

```diff
@@ -54,7 +54,7 @@ The following observations can be made about such applications:
 - They may be written with no explicit knowledge of the hardware capabilities of the target
    platform, such as whether an HSM or TPM is available.
 - They are often sharing the target platform hardware with other applications due to the use of
-   virtualization or containerization technology.
+   virtualisation or containerization technology.
```

Comment:
if you change virtualisation, then why not containerisation?

Reply:
Ugh, it seems there are a lot more to change... Not sure this is the right place for this.

Reply:
I think if there are any changes in the British/USA english then it would merit a different PR. Also, I think we should default to the USA english.